source/features/quick-mention.tsx

```diff
@@ -50,34 +53,51 @@ function add(avatar: HTMLElement): void {
 		// Reviews
 		'.js-comment',
 	])!;
-	if (debug) {
-		timelineItem.style.border = 'solid 5px red';
-	}
 
-	if (
-		// TODO: Rewrite with :has()
-		// Exclude events that aren't tall enough, like hidden comments or reviews without comments
-		!elementExists('.unminimized-comment, .js-comment-container', timelineItem)
-	) {
-		return;
+	if (timelineItem) {
+		// Old view
+		if (debug) {
+			timelineItem.style.border = 'solid 5px red';
+		}
+
+		if (
+			// TODO: Rewrite with :has()
+			// Exclude events that aren't tall enough, like hidden comments or reviews without comments
+			!elementExists('.unminimized-comment, .js-comment-container', timelineItem)
+		)
+			return;
+	} else {
+		// Make sure the comment isn't hidden
+		const contentItem = avatar.parentElement!.querySelector('[data-testid="comment-header"] + div')!;
+
+		if (!contentItem) {
+			return;
+		}
 	}
 
 	if (debug) {
 		timelineItem.style.border = 'solid 5px green';
 	}
 
+	const isOldView = !avatar.hasAttribute('data-testid');
```

Comment:
Line 58 appears to suggest that only the old view has a specific element, so you can use the same detection here instead of a new detection

```suggestion
	const isOldView = Boolean(timelineItem);
```

I'd also move this line above the first condition so you can delete the comment on line 58

Reply:
By the way, if the differences between two views are more than a couple of selectors, it's sometimes easier to copy-paste the whole `init` and customize it properly. When the time comes, I can just delete the old `init` instead of walking through each condition. This method also makes it easier to write the new version. 

Not necessary though in this PR though